### ISSUE TITLE
[BACK_END] Choosing Algorithm

### ISSUE THREAD

**Issue by joseph280** [meltingpotteam/geeft#11]
Description: Programation of the choosing algorithm and attaching it to its respective button.
Requires: Button implementation, algorithm testing.
Status: pseudo algorithm already done, remains its implementation inside the server.
Time: 8 hours.

**Comment by danybr-dev** [comment]
First part of algorithm is done: now we can select users reserved to an ad in the app after the deadline

**Comment by joseph280** [comment]
Don't get it. What do you mean? You already implement it with the code I write
or no?

**Comment by joseph280** [comment]
All working as expected.
